src/app/SimpleDownloadDelegate.qml

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2015-2016 Canonical Ltd.
+ *
+ * This file is part of morph-browser.
+ *
+ * morph-browser is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; version 3.
+ *
+ * morph-browser is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+import QtQuick 2.9
+import QtWebEngine 1.10
+import Ubuntu.Components 1.3
+
+ListItem {
+    id: simpleDownloadDelegate
+
+    property var download
+    property alias title: listItemlayout.title
+    property alias subtitle: listItemlayout.subtitle
+    property string icon
+    property alias image: thumbimage.source
+
+    readonly property bool incomplete: !download.isFinished
+    readonly property bool error: download.interruptReason > WebEngineDownloadItem.NoReason
+    readonly property bool cancelled: download.interruptReason == WebEngineDownloadItem.UserCanceled
+    readonly property bool paused: download.isPaused
+    readonly property int progress: download ? 100 * (download.receivedBytes / download.totalBytes) : -1
+    
+    signal cancel()
+    signal remove()
+    signal deleted()
+    
+    divider.visible: false
+
+    MimeData {
+        id: linkMimeData
+        
+        text: download.url
+    }
+    
+    ListItemLayout {
```

Comment:
It doesn't appear that you can tap on downloads in this view to open them. I tried with a PDF file. Is that expected?

Reply:
Apparently, I commented out the codes for this because it needs #414 to be merged first. It uses the same popup :sweat_smile: